src/main/java/ch/njol/skript/expressions/ExprDurability.java

```diff
@@ -73,45 +85,67 @@ public String getPropertyName() {
 	@Override
 	@Nullable
 	public Class<?>[] acceptChange(final ChangeMode mode) {
-		if (mode == ChangeMode.REMOVE_ALL)
-			return null;
-		if (Slot.class.isAssignableFrom(getExpr().getReturnType()) || getExpr().isSingle() && ChangerUtils.acceptsChange(getExpr(), ChangeMode.SET, ItemStack.class, ItemType.class))
-			return CollectionUtils.array(Number.class);
+		switch (mode) {
```

Comment:
For this one, I think it's fine to do 
```
if (mode == ChangeMode.DELETE)
	return CollectionUtils.array(Number.class);
return null
```
Because the addition of future ChangeModes are not affected.

Reply:
Why won't I only return number.class if the mode == Delete? What about the other modes?

Reply:
Oops, I misread, yeah, that's the right way to do it then.